api/tests/Feature/ExperienceTest.php

```diff
@@ -474,4 +477,45 @@ public function testSkillRelationshipSkipsSoftDeletedPivots(): void
         // assert that the soft-deleted relationship is ignored
         $this->assertCount(2, $experience->fresh()->skills);
     }
+
+    public function testExperiencesFetchSoftDeletedSkills(): void
+    {
+        // create experience with three skills, then soft delete a skill
+        Skill::factory()->count(3)->create();
+        $experience = AwardExperience::factory()->withSkills(3)->create([
+            'user_id' => $this->platformAdmin->id,
+        ]);
+        $randomDeletedSkill = Skill::first();
+        DB::table('skills')
+            ->where('id', $randomDeletedSkill->id)
+            ->update([
+                'deleted_at' => config('constants.past_datetime'),
+            ]);
```

Comment:
This isn't wrong, but you could just use `->delete()`

Reply:
Ah, I don't need the Facade at all here, I can use the other version of `delete()`
Done in https://github.com/GCTC-NTGC/gc-digital-talent/pull/7762/commits/a79de8ee608fd8973167be359ee6dfa661711581